manifests/postgresql.crd.yaml

```diff
@@ -375,6 +375,10 @@ spec:
                 properties:
                   s3_wal_path:
```

Comment:
keeping is `required` in case some users depend on that

Reply:
This can be dropped. Doesn't make sense if we give host/port precedence over wal path. Will not break existing clusters.